docs/Configuration.md

```diff
@@ -54,6 +54,11 @@ Will create a datadir of `~/.bcoin_spv`, containing a chain database, wallet dat
 - `cache-size`: Size (in MB) of leveldb cache and write buffer (default: 32mb).
 - `spv`: Enable Simplified Payments Verification (SPV) mode
 
+*Note: The `spv` option can not be entered in `bcoin.conf`. It will only work when passed as a launch argument:*
```

Comment:
Maybe we should mention the same for `--daemon` while we're here.

Reply:
👍